src/index.js

```diff
@@ -1,6 +1,15 @@
 import fullpage from "fullpage.js";
 import "./app.scss";
 import "./style.scss";
+import randomWords from "random-words";
+
+document.querySelectorAll(".randomText").forEach(e => {
+  e.addEventListener("click", e => {
+    e.target.innerHTML = randomWords({ min: 3, max: 100 }).reduce(
+      (a, b) => `${a} ${b}`
+    );
+  });
+});
```

Comment:
Again you can bind `window` or `document.body` rather than iterating all nodes.

Reply:
Like this?
```
window.addEventListener("click", e => {
  if (e.target.classList.contains("randomText")) {
    e.target.innerHTML = randomWords({ min: 3, max: 100 }).reduce(
      (a, b) => `${a} ${b}`
    );
  }
});
```

Reply:
yea

Reply:
Out of curiosity, is that make it more efficiency since browser won't need to add that many `listener` on node?

Reply:
yes. when you bind event you basically store the event type and event handler mapping in an array of objects (Check the implementation of EventEmitter). It definitely occupies for more memory. When you bind hundreds of elements with their event handlers, you will easily see the performance going down. 

Another benefit of using event bubbling is you don't need to worry if an element is rendered or removed. It's more reactive.